src/NodaTime/Calendars/Wondrous/WondrousCalendarHelper.cs

```diff
@@ -0,0 +1,2204 @@
+                case 663:
+                    return new WondrousYearInfo(4, 21);
+                case 664:
+                    return new WondrousYearInfo(4, 21);
+                case 665:
+                    return new WondrousYearInfo(5, 20);
+                case 666:
+                    return new WondrousYearInfo(4, 21);
+                case 667:
+                    return new WondrousYearInfo(4, 21);
+                case 668:
+                    return new WondrousYearInfo(4, 21);
+                case 669:
+                    return new WondrousYearInfo(5, 20);
+                case 670:
+                    return new WondrousYearInfo(4, 21);
+                case 671:
+                    return new WondrousYearInfo(4, 21);
+                case 672:
+                    return new WondrousYearInfo(4, 21);
+                case 673:
+                    return new WondrousYearInfo(4, 20);
+                case 674:
+                    return new WondrousYearInfo(5, 20);
+                case 675:
+                    return new WondrousYearInfo(4, 21);
+                case 676:
+                    return new WondrousYearInfo(4, 21);
+                case 677:
+                    return new WondrousYearInfo(4, 20);
+                case 678:
+                    return new WondrousYearInfo(5, 20);
+                case 679:
+                    return new WondrousYearInfo(4, 21);
+                case 680:
+                    return new WondrousYearInfo(4, 21);
+                case 681:
+                    return new WondrousYearInfo(4, 20);
+                case 682:
+                    return new WondrousYearInfo(5, 20);
+                case 683:
+                    return new WondrousYearInfo(4, 21);
+                case 684:
+                    return new WondrousYearInfo(4, 21);
+                case 685:
+                    return new WondrousYearInfo(4, 20);
+                case 686:
+                    return new WondrousYearInfo(5, 20);
+                case 687:
+                    return new WondrousYearInfo(4, 21);
+                case 688:
+                    return new WondrousYearInfo(4, 21);
+                case 689:
+                    return new WondrousYearInfo(4, 20);
+                case 690:
+                    return new WondrousYearInfo(5, 20);
+                case 691:
+                    return new WondrousYearInfo(4, 21);
+                case 692:
+                    return new WondrousYearInfo(4, 21);
+                case 693:
+                    return new WondrousYearInfo(4, 20);
+                case 694:
+                    return new WondrousYearInfo(5, 20);
+                case 695:
+                    return new WondrousYearInfo(4, 21);
+                case 696:
+                    return new WondrousYearInfo(4, 21);
+                case 697:
+                    return new WondrousYearInfo(4, 20);
+                case 698:
+             
```

Comment:
I'd expect all this information and the method itself to be in the WondrousCalculator. I wouldn't expect it to be needed outside that, except maybe for testing.